cves/CVE-2009-1195.yml

```diff
@@ -90,7 +96,10 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer:
+  answer: |
+    doesn't look like there are any unit tests, its hard to tell because
+    it is a config issue so there are multiple files and I can't find specific
+    tests for any of them, also their test naming convetions are a bit confusing
   code:
   fix:
```

Comment:
The fix could be add a unit test and use easier to comprehend naming conventions

Reply:
`For the "fix" answer below, check if the fix for the vulnerability involves
    adding or improving an automated test to ensure this doesn't happen again.` it is supposed to refer to the fix done not a possible fix